Gemfile

```diff
@@ -9,17 +7,30 @@ end
 
 gem 'puma', '~> 3.7'
 gem 'rails', '~> 5.1.3'
+gem 'sidekiq'
```

Comment:
I'm using sidekiq here to store the hit from github into my rails application, i receive it and process it eventually, this way i keep track of each request sent to me from the integration API